lib/hooks/useUserCapabilities/useUserCapabilities.js

```diff
@@ -0,0 +1,82 @@
+import {
+  pick,
+  mapValues,
+  keyBy,
+} from 'lodash';
+import { useMemo } from 'react';
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+  useStripes,
+} from '@folio/stripes/core';
+
+import { CAPABILITIES_LIMIT } from '../../constants';
+import { getCapabilitiesGroupedByTypeAndResource } from '../../utils';
+
+/**
+ * Get capabilities for a given user.
+ *
+ * @param {string} userId The user ID.
+ * @param {string} tenant The Tenant ID. Passes into `useOkapiKy` which will default to `stripes.okapi.tenant` if omitted.
+ * @param {string} roleId The Role ID
+ * @param {boolean} expand Defines if capability sets must be expanded in the API response. Defaults to `false`.
+ * @param {object} options Any additional options to pass into `useQuery()`.
+ * @returns Capabilities.
+ */
+export const useUserCapabilities = (userId, tenant = '', roleId, expand = false, options = {}) => {
```

Comment:
I was thinking if we could use an object as a params if it is more than 3. 
Ex: `{ userId, tenant = '', roleId, expand = false, options = {} })`